src/app/header/header.component.html

```diff
@@ -79,7 +79,7 @@
               </a>
             </li>
             <li class="divider"></li>
-            <li>
+            <li id="createSpace">
```

Comment:
These IDs should have a prefix of some kind since the code gets compiled together, we don't want any duplicate IDs. I suggest something like "header-createSpace" or "headerCreateSpace".